installer/conf/logrotate.conf

```diff
@@ -6,3 +6,12 @@
     size 100M
     copytruncate
 }
+
+/var/opt/microsoft/omsagent/log/update_run_progress.log {
```

Comment:
I guess you used /%WORKSPACE_ID%/ in the path in last iteration, just curious why are we not using workspace specific path